mteb/tasks/Classification/kor/KlueTC.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class KlueTC(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="KLUE-TC",
+        dataset={
+            "path": "klue",
+            "name": "ynat",
```

Comment:
```suggestion
           
```
Remove since there is no subset in this dataset

Reply:
KLUE is a collection of subtasks created by multiple organizations, so YNAT exists as a subset of KLUE. See the comment above. ;)